ReactSkia/views/common/RSkImageCacheManager.h

```diff
@@ -13,20 +13,32 @@
 #define SKIA_CPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 #define SKIA_GPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 
+using namespace std;
+
 namespace facebook {
 namespace react {
+namespace ImageCacheManager {
```

Comment:
both namespace and class name is representing ImageCacheManager 
Is it required ?
Option-1 : Remove the "ImageCacheManager" namespace
Option-2 : Name the namespace as CacheManager